concreate/descriptor.py

```diff
@@ -309,52 +309,3 @@ def _copy_impl(self, target):
         return target
 
 
-def merge_dictionaries(dict1, dict2):
```

Comment:
I hope there are no changes here except for moving it to different place - it hard to review otherwise

Reply:
There are, I commented on it here: https://github.com/jboss-container-images/concreate/issues/32#issuecomment-329460848